code/turf/floors.dm

```diff
@@ -1477,11 +1478,17 @@ TYPEINFO(/turf/simulated/floor/grass)
 	overlay.blend_mode = BLEND_ADD
 	overlay.alpha = alpha
 	overlay.color = color
+
+	if (istype(src, /turf/simulated/floor/airless/plating/catwalk)) // "Guh" - Leah
+		catwalk = locate() in src
+		catwalk.UpdateOverlays(overlay, "wet_overlay")
+
 	src.UpdateOverlays(overlay, "wet_overlay")
 	src.wet = wetType
 
 	SPAWN(timeout)
 		src.UpdateOverlays(null, "wet_overlay")
+		catwalk.UpdateOverlays(null, "wet_overlay")
```

Comment:
```suggestion
		catwalk?.UpdateOverlays(null, "wet_overlay")
```
Oh actually it needs this because `catwalk` is going to be null most of the time